src/pybind/mgr/dashboard/services/custom_banner.py

```diff
@@ -0,0 +1,11 @@
+import logging
+
+from .. import mgr
+
+logger = logging.getLogger(__name__)
+
+
+def get_custom_login_banner_file():
+    file = mgr.get_store('custom_login_banner')
+    logger.info(f'Reading login banner file: {file}')
+    return file
```

Comment:
Nice isolation (controller - service)!

Given that the get-set methods could be both seen as services, why not moving them to here, instead of "bloating" the module.py file with more code?

Reply:
That was the initial plan, but on placing the CLICommand in `src/pybind/mgr/dashboard/services/custom_banner.py`, the ceph cli was unable to find that command and gave a `Invalid Command` error. Anyways have placed the base function (used by cli and api) in `custom_banner.py` and the cli-command in the `module.py`